torchvision/datasets/cifar.py

```diff
@@ -172,3 +172,7 @@ class CIFAR100(CIFAR10):
     test_list = [
         ['test', 'f0ef6b0ae62326f3e7ffdfab6717acfc'],
     ]
+    """`CIFAR100 <https://www.cs.toronto.edu/~kriz/cifar.html>`_ Dataset.
```

Comment:
This docstring needs to be right under the class definition for sphinx to display it properly i.e. it should look like:
```python
class CIFAR100(CIFAR10):
"""`CIFAR100 <....>
This is a subclas ...
"""
```

Reply:
Got it, thanks!